docs/skopeo.1.md

```diff
@@ -60,6 +60,10 @@ Uses the system's trust policy to validate images, rejects images not trusted by
 
   _destination-image_ use the "image name" format described above
 
+  **--compress, -c** _bool-value_ Option to compress blobs of image when saving image to directory using the 'dir:' transport (default is false)
```

Comment:
I hate the work blobs.  Means nothing to me.  

Should force the compression of  tarball image layers when saving to directory using the 'dir' transport.  (default is to store the images the same they the source)

Which brings up a point, should we have an  --uncompress option?